### PR TITLE
[MDU-21] IE fixes for icons and DonnutChart component

### PR DESCRIPTION
**Pull request by diogobeda**
### This PR changes/includes the following components:

- Many icons
- DonutCharts
- Iso

### This PR includes:

- [ ] Unit tests for every change/new component
- [ ] Documentation for every change/new component

### Does this PR has breaking changes?

No

### CODE REVIEW

File: components/icons/IconActivity.js [hunk 1]
```diff
@@ -2,33 +2,34 @@ import React from "react"
...
+        width: 24,
+        height: 24,
+        className: null
     }
 
-
     render() {
-
-        let cn = className("m-icon-activity", "m-icon-path",this.props.className)
+        let cn = className(
```
Comment (jimmed) [inline components/icons/IconActivity.js]: Can be a `const`!

File: components/icons/IconActivity.js [hunk 2]
```diff
@@ -2,33 +2,34 @@ import React from "react"
...
+                height={this.props.height}
+                viewBox="0 0 24 24"
+                className={cn}
+            >
+                <g fill="none" fillRule="evenodd">
+                    <path d="M7 11.5h10" />
+                    <path
+                        fillRule="nonzero"
+                        d="M5.5 13a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3zm0 1a2.5 2.5 0 1 1 0-5 2.5 2.5 0 0 1 0 5zM18.5 13a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3zm0 1a2.5 2.5 0 1 1 0-5 2.5 2.5 0 0 1 0 5z"
+                    />
```
Comment (jimmed) [inline components/icons/IconActivity.js]: TODO, but more generally as a separate issue:

Comment (diogobeda) [reply components/icons/IconActivity.js]: For some cases, I like component icons, though. It allows for a level flexibility where static assets don't. But I agree that most of the cases a static svg is better

Comment (jimmed) [reply components/icons/IconActivity.js]: Many, many moons ago, I worked on a large React app - one of my colleagues at the time created a webpack loader that imported an SVG file and spat out a React component. I'm not sure he ever released it to the public, but we could easily replicate it later on!

**Pull request merged**